test/Datadog.Trace.ClrProfiler.IntegrationTests/SmokeTests/SmokeTestBase.cs

```diff
@@ -81,6 +83,13 @@ protected void CheckForSmoke()
 
                 var ranToCompletion = process.WaitForExit(MaxTestRunMilliseconds);
 
+                if (AssumeSuccessOnTimeout && !ranToCompletion)
+                {
+                    process.Kill();
```

Comment:
For some reason my reproduction is leaving unresolved Tasks around in CI. It might be the way I'm starting all the requirements for Orleans in the sample.